app/1.0.x/proxy.md

```diff
@@ -925,14 +925,13 @@ the `cert.pem` certificate previously configured.
 
 [Back to TOC](#table-of-contents)
 
-### Restricting the client protocol (HTTP/HTTPS)
+### Restricting the client protocol (HTTP/HTTPS/TCP/TLS)
 
 Routes have a `protocols` property to restrict the client protocol they should
-listen for. This attribute accepts a set of values, which can be `http` or
-`https`.
+listen for. This attribute accepts a set of values, which can be `http`,
+`https`, `tcp` & `tls`.
```

Comment:
`&` => `or`

Formal English writing avoids the use of `&` (I noticed you do this quite a bit: is the rule different is Spanish? :) )

Reply:
I honestly ignored that rule. I thought they were interchangeable. Thanks for teaching me something today!